.gitignore

```diff
@@ -29,3 +29,5 @@ __pycache__
 /tests/.vscode
 /sonobuoy-output
 *.tmp
+/.vscode
```

Comment:
This should be in your local gitignore

Reply:
:heavy_check_mark: 
Didn't even know there was such a thing :D